src/app/video-games/detail/components/video-game-detail/video-game-detail.component.scss

```diff
@@ -9,4 +10,53 @@
   h2, p {
     margin-top: 0;
   }
-}
\ No newline at end of file
+}
+
+.btn-group {
+  padding: 10px 20px;
```

Comment:
When the button group component is created, these styles can go into the host for the component